components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -748,14 +748,15 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             await this.guardAccess({ kind: "workspace", subject: workspace }, "get");
         }
 
-        this.internalStopWorkspace(ctx, workspace).catch((err) => {
+        this.internalStopWorkspace(ctx, workspace, "stopped via API").catch((err) => {
             log.error(logCtx, "stopWorkspace error: ", err);
         });
     }
 
     protected async internalStopWorkspace(
         ctx: TraceContext,
         workspace: Workspace,
+        reason: string,
```

Comment:
:cloud: Maybe this should be optional? :thinking:

Reply:
Why make it optional if we always want to know the reason a workspace was stopped?